frontend/helpers/airtable.tsx

```diff
@@ -73,3 +90,34 @@ export const getRoommatesData = async () => {
   queryResult.unloadData();
   return roommatesData;
 };
+
+export const getTransactionsHistoryData = async () => {
+  let queryResult;
+  try {
+    const transactionsHistoryTable = base.getTable(HISTORY_TABLE_ID);
+    queryResult = await transactionsHistoryTable.selectRecordsAsync();
+
+    const historiesData = queryResult.records.map(record => ({
+      date: record.getCellValue(HISTORY_DATE_FIELD_ID),
+      RUM: record.getCellValue(HISTORY_RUM_FIELD_ID)
+    }));
+  
+  return historiesData;
+  } catch (e) {
+    addMessageAndThrow(e, 'error during extraction of history table');
+  } finally {
+    if (queryResult && queryResult.isDataLoaded) queryResult.unloadData();
+  }
+};
+
+export const addRecords = async (tableId, data) => {
+  let table;
+  try {
+    table = base.getTable(tableId);
+
+    const records = lodash.chunk(data, BATCH_SIZE);
+    await Promise.all(records.map(async record => await table.createRecordsAsync(record)));
+  } catch (e) {
+    addMessageAndThrow(e, `error during creation of ${table.name} table`);
```

Comment:
"error during records creation in ${table.name} table"

je suis pas fan de table.name :  car si il y a un bug dans la methode `base.getTable` plus haut, `table.name` ne sera pas defini et on aura une erreur dans un catch.

Reply:
up

Reply:
done